src/libsync/account.cpp

```diff
@@ -84,7 +84,9 @@ Account::~Account() = default;
 
 QString Account::davPath() const
 {
-    return QLatin1String("/remote.php/dav/files/") + davUser() + QLatin1Char('/');
+    const auto davPathBase = QLatin1String("/remote.php/dav/files/");
+    const auto davPathFinal = !davUser().isEmpty() ? QString(davPathBase + davUser()) + QLatin1Char('/') : QString(davPathBase);
```

Comment:
QString(davPathBase) is redundant or?

Reply:
@FlexW Conversion is here just to avoid a problem we had in past when returning `QStringBuilder` instead of `QString`, or, I did not get your comment ?